### PR TITLE
fix some typos found during translations

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

File: pages/translations/ar/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1313,11 +1313,11 @@ msgid "Shaun Zacharia, President & Co-Founder, Triplelift."
...
+msgid "The average mobile site takes 19 seconds to load over a 3G connection. Coupled with the fact that conversions fall by 12% for every additional second a webpage takes to load, you can safely assume two things:  speed equals performance, and most advertiser campaigns have room to improve. That’s where you come in. As an SSP, DSP, analytics provider, or anything in between, embracing speed means improving campaign performance, delivering quality user experiences, and establishing yourself as the adtech platform for the fast web. Join the hundreds of technology platforms currently supporting the open source AMP ecosystem. "
```
Comment (CrystalOnScript) [inline pages/translations/ar/LC_MESSAGES/messages.po]: Looks like theres two spaces here `things:  speed`

Comment (CrystalOnScript) [reply pages/translations/ar/LC_MESSAGES/messages.po]: Also a space here: `ecosystem. "`

File: pages/translations/ar/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -1313,11 +1313,11 @@ msgid "Shaun Zacharia, President & Co-Founder, Triplelift."
...
 msgstr ""
 
 #: pages/content/amp-dev/about/ads.html:209
-msgid "AMP offers a way for you to give users a faster experience everywhere  on ads, landing pages or on your entire website. AMP ads help users engage with your brand and improve the performance of your campaigns. When your target audience lands on an AMP page, they get what they’re looking for in the moment they’re looking for it. The net result – they’re more likely to be engaged with and receptive to your brand’s messages, and more likely to take the actions you’re guiding them towards like completing a purchase."
+msgid "AMP offers a way for you to give users a faster experience everywhere: on ads, landing pages or on your entire website. AMP ads help users engage with your brand and improve the performance of your campaigns. When your target audience lands on an AMP page, they get what they’re looking for in the moment they’re looking for it. The net result – they’re more likely to be engaged with and receptive to your brand’s messages, and more likely to take the actions you’re guiding them towards like completing a purchase."
```
Comment (CrystalOnScript) [inline pages/translations/ar/LC_MESSAGES/messages.po]: I found all of these words concerning and terrible. I tried rewriting it but its kind of a mess. 

High level question: do we want to spend time with these other about pages before getting them translated?

File: pages/translations/ar/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -1581,7 +1581,7 @@ msgid "Prioritize resource loading"
 msgstr ""
 
 #: pages/content/amp-dev/about/how-amp-works.html:85
-msgid "AMP controls all resource downloads  it prioritizes resource loading, loading only what’s needed, and prefetches lazy-loaded resources."
+msgid "AMP controls all resource downloads: it prioritizes resource loading, loading only what’s needed, and prefetches lazy-loaded resources."
```
Comment (CrystalOnScript) [inline pages/translations/ar/LC_MESSAGES/messages.po]: This sentence is also awful. It says uses the word `resources` and `loading` three times.

File: pages/translations/ar/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -2004,7 +2004,7 @@ msgstr ""
 msgid "Used by popular and global platforms like Google, Bing and Twitter, AMP allows you to ensure users from all these surfaces get an unparalleled, often instantaneous and native-feeling experience by defaulting to AMP pages when available."
 msgstr ""
 
-msgid "You can use CSS to customize your styling, dynamic data to fetch the freshest data where needed, to build the best possible userexperience for your customers."
+msgid "You can use CSS to customize your styling, dynamic data to fetch the freshest data where needed, to build the best possible user experience for your customers."
```
Comment (CrystalOnScript) [inline pages/translations/ar/LC_MESSAGES/messages.po]: I don't think this sentence makes much sense.

File: pages/translations/ar/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -2239,7 +2239,7 @@ msgstr ""
...
+msgid "The average mobile site takes 19 seconds to load over a 3G connection. Coupled with the fact that conversions fall by 12% for every additional second a webpage takes to load, you can safely assume two things:: speed equals performance, and most advertiser campaigns have room to improve. That’s where you come in. As an SSP, DSP, analytics provider, or anything in between, embracing speed means improving campaign performance, delivering quality user experiences, and establishing yourself as the adtech platform for the fast web. Join the hundreds of technology platforms currently supporting the open source AMP ecosystem. "
```
Comment (CrystalOnScript) [inline pages/translations/ar/LC_MESSAGES/messages.po]: Theres now a double `::` here:  `two things:: `

Is it `adtech` or `ad tech`?

File: pages/translations/en/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -1101,7 +1101,7 @@ msgid "The AMP project strongly encourages your participation and contributions!
 msgstr ""
 
 #: frontend/templates/views/detail/component-detail.j2:33
-msgid "You've read this document a dozen times but it doesn't really cover all of your questions? Maybe other people felt the same  reach out to them on Stack Overflow."
+msgid "You've read this document a dozen times but it doesn't really cover all of your questions? Maybe other people felt the same:  reach out to them on Stack Overflow."
```
Comment (CrystalOnScript) [inline pages/translations/en/LC_MESSAGES/messages.po]: another double space: `same:  reach`

File: pages/translations/en/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -1381,11 +1381,11 @@ msgid "Shaun Zacharia, President & Co-Founder, Triplelift."
...
+msgid "The average mobile site takes 19 seconds to load over a 3G connection. Coupled with the fact that conversions fall by 12% for every additional second a webpage takes to load, you can safely assume two things:  speed equals performance, and most advertiser campaigns have room to improve. That’s where you come in. As an SSP, DSP, analytics provider, or anything in between, embracing speed means improving campaign performance, delivering quality user experiences, and establishing yourself as the adtech platform for the fast web. Join the hundreds of technology platforms currently supporting the open source AMP ecosystem. "
```
Comment (CrystalOnScript) [inline pages/translations/en/LC_MESSAGES/messages.po]: Space: `ecosystem. "`

File: pages/translations/es/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -1057,7 +1057,7 @@ msgid "The AMP project strongly encourages your participation and contributions!
 msgstr ""
 
 #: frontend/templates/views/detail/component-detail.j2:33
-msgid "You've read this document a dozen times but it doesn't really cover all of your questions? Maybe other people felt the same  reach out to them on Stack Overflow."
+msgid "You've read this document a dozen times but it doesn't really cover all of your questions? Maybe other people felt the same:  reach out to them on Stack Overflow."
```
Comment (CrystalOnScript) [inline pages/translations/es/LC_MESSAGES/messages.po]: Double space: `same:  reach`

File: pages/translations/ar/LC_MESSAGES/messages.po (hunk 1 above)
Comment (patrickkettner) [reply pages/translations/ar/LC_MESSAGES/messages.po]: Ah, good catch. That is actually how it is in the html itself. I'll update all locations

File: pages/translations/ar/LC_MESSAGES/messages.po (hunk 2 above)
Comment (patrickkettner) [reply pages/translations/ar/LC_MESSAGES/messages.po]: im assuming that would occur after the redesign occurs? But yeah, anytime.

File: pages/translations/ar/LC_MESSAGES/messages.po (hunk 3 above)
Comment (patrickkettner) [reply pages/translations/ar/LC_MESSAGES/messages.po]: 😬

File: pages/translations/ar/LC_MESSAGES/messages.po (hunk 5 above)
Comment (patrickkettner) [reply pages/translations/ar/LC_MESSAGES/messages.po]: `adtech` is a thing, its how I usually see it written. not sure if there is a preferred way

**Pull request merged**